### ISSUE TITLE
Code example for SPO2

### ISSUE THREAD

**Issue by r3dh0t** [pimoroni/max30105-python#4]
It would be fantastic to have a code example for SPO2 readings in Python.

An arduino example is here: https://github.com/sparkfun/SparkFun_MAX3010x_Sensor_Library/blob/master/examples/Example8_SPO2/Example8_SPO2.ino

**Comment by iioaia** [comment]
Hi I would like to learn to program hardware in this way, grab a piece of
hardware with little or no python library and create one.

To get the most out of the sensors I buy.

 What is that type of programming called? I'd like to expand the max30105
library or examples with that knowledge.

Thanks!

**Comment by r3dh0t** [comment]
I'm not sure if I understand sorry @iioaia. Are you wanting to create the example?

Ideally, I'd just like to be able to return the SPO2 values from the sensor with Python. The current example can print HR data, but not SPO2 data.
https://github.com/pimoroni/max30105-python/tree/master/examples

**Comment by hackebike** [comment]
Looking for the SPO2 reading too, not sure is implemented in the Class, any info?

**Comment by MichielBontenbal** [comment]
This is very sad :-( I bought this device especially for the SpO2 sensor and then there is no script!

**Comment by ilan-schemoul** [comment]
Yeah I'm also looking forward for this. I feel like marketplaces should not say it's raspberry compatible as the software is not there yet.
Anyway thanks a lot for this amazing library.
From my understanding it's not just the example that lacks but also the code ? And as the code to compute oxygen is complex one should probably do a binding to that https://github.com/sparkfun/SparkFun_MAX3010x_Sensor_Library/blob/master/src/spo2_algorithm.cpp ? right ?

**Comment by Gadgetoid** [comment]
I'm very, very late to the party here but I've added a Python port of the MAX3010x SPO2 algorithm as part of #12
